sys/key_rev_archival_value.js

```diff
@@ -0,0 +1,149 @@
+"use strict";
+
+var P = require('bluebird');
+var uuid = require('cassandra-uuid').TimeUuid;
+var preq = require('preq');
+
+var HyperSwitch = require('hyperswitch');
+var HTTPError = HyperSwitch.HTTPError;
+var URI = HyperSwitch.URI;
+
+var mwUtil = require('../lib/mwUtil');
+
+var spec = HyperSwitch.utils.loadSpec(__dirname + '/key_rev_value.yaml');
+
+function ArchivalBucket(options) {
+}
+
+ArchivalBucket.prototype._latestName = function(bucket) {
+    return bucket + '.latest';
+};
+
+ArchivalBucket.prototype._archiveName = function(bucket) {
+    return bucket + '.archive';
```

Comment:
Yeah, as long as we don't create any new buckets it works out the same. With the explicit alias setup, new tables could start using the new suffixes right away, saving a few keyspace renames with possibly associated downtime later.